### PR TITLE
Tidy up Place page

### LINKED ISSUE #51: Tidy up styling of Place page
The MVP place page is a bit hard to parse:

overview

### PR DESCRIPTION
**Pull request by zarino**
screen shot 2017-02-17 at 13 05 23

Fixes #51.

### CODE REVIEW

**Comment by octopusinvitro** [comment]
Looks good to me :+1:

The only thing I am not sure about is removing the navigation pills, as I don't know if we are ever going to implement what should appear under `Places`... I will left @mhl decide on that

**Comment by mhl** [comment]
:+1: Looks great, thank-you @zarino. At some point we'll want to use leaflet to render the boundary of the area, but could nicely go over the big map pin, I suppose.
